src/ant/src/AntennaChecker.cc

```diff
@@ -539,12 +550,7 @@ bool AntennaChecker::checkPAR(odb::dbTechLayer* tech_layer,
           info.PAR,
           PAR_ratio,
           violation ? "(VIOLATED)" : "");
-      if (verbose) {
-        logger_->report("{}", par_report);
-      }
-      if (report_file.is_open() && (violation || verbose)) {
-        report_file << par_report << "\n";
-      }
+      net_to_report_[db_net].report += par_report + "\n";
```

Comment:
As it required for MT that db_net be present it would be better to use `net_to_report_.at(db_net)` to enforce it.  Likewise elsewhere.